charts/operator/templates/certmanager/serving-cert-certificate.yaml

```diff
@@ -0,0 +1,15 @@
+{{ if and .Values.certManager.enabled (not .Values.webhooks.useCustomTlsCerts) .Values.global.operator.enabled -}}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name }}-airflow-operator-serving-cert
+  namespace: '{{ .Release.Namespace }}'
+spec:
+  dnsNames:
+  - airflow-operator-webhook-service.{{ .Release.Namespace }}.svc
+  - airflow-operator-webhook-service.{{ .Release.Namespace }}.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: airflow-operator-selfsigned-issuer
```

Comment:
this should be {{ .Release.Name }}-airflow-operator-selfsigned-issuer

Reply:
done